pulser/_seq_drawer.py

```diff
@@ -275,9 +295,14 @@ def phase_str(phi: float) -> str:
             cs_detuning = CubicSpline(solver_time, yb2)
             yaeff = cs_amp(teff)
             ybeff = cs_detuning(teff)
+        if draw_modulation and (ch_obj.mod_bandwidth or not draw_input):
+            t_diffs = np.diff(times)
+            input_a = np.repeat(ya[1:], t_diffs)
+            input_b = np.repeat(yb[1:], t_diffs)
```

Comment:
Is it necessary to `repeat`, since `plot` will already relate points with segments?

Reply:
Yes, because these values will be modulated. When this is not the case, they are not repeated and are plotted as you suggest